.github/workflows/node.js.yml

```diff
@@ -53,24 +57,33 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-lintcache-
 
+      - name: Cache node modules
+        id: cache-node-modules
+        uses: actions/cache@v3
+        with:
+          path: |
+            node_modules
+            packages/*/node_modules
```

Comment:
How big is this cache? I think there's a cache explorer now for github actions. We might want to ignore something like `node_modules/.cache` if it's too big. Although I don't think it should be that big since we're no longer using webpack and terser which create huge (1GB+) caches

This is 1 area that yarn (with PnP mode) or pnpm have some advantages. They don't extract the package contents so there's significantly less file overhead. Only a few hundred files instead of (often) tens of thousands. But that would be a completely different ticket and maybe not worth changing package managers if they cause a headache

Reply:
This cache is ~420MB:
```
node-modules-a852c260fe0187e618cdb16aaed716abc473d2fa7a857c0f7695f7b489a0e629
420 MB cached 10 hours ago
```
It saves ~1 minute to use a cache instead of doing an `npm ci` each time.